lintrules/draftcheck/script.py

```diff
@@ -32,6 +32,14 @@ def print_warning(fname, lineno, line, span, rule, args):
     print(prefix, end=' ')
     print(f"\t[{rule.id:03d}]", get_brief(rule))
 
+def get_no_comment_line(line):
+    n = len(line)
+    for i in range(1, n):
+        if line[i] == '%':
+            if line[i-1] == "\\":
+                continue
+            return line[:i]
+    return line
```

Comment:
```suggestion
def remove_latex_comment(line):
    for i in range(len(line)):
        if line[i] == '%':
            if i > 0 and line[i - 1] == "\\":
                continue
            return line[:i]
    return line
```
こんな感じにしたら line[0] == '%' と書かなくて良さそうだと思いました！どうでしょうか？

Reply:
そっちのほうが綺麗ですね！
変えてチェックしておきます！